### DISCUSSION TITLE
How to open files with spaces in the name

### DISCUSSION THREAD

**Discussion by ChanManChan** [gokcehan/lf#1647] [Q&A] [answered]
![2024-03-18-125137](https://github.com/gokcehan/lf/assets/42992965/5c229cf1-4810-4c3c-ae1a-7932485089d2)
[top segment is lfrc and bottom is a lf instance]

How do I open a file with spaces in its name? Every other file opens without a fault using the xdg-open command except for the files named similar to the highlighted one in the picture.

**Comment by DusanLesan** [comment, marked as answer]
I have solved that issue by having `set ifs "\n"` in top of my lfrc

**Reply by ChanManChan** [reply]
Thanks a lot, it worked ! Do you mind explaining how it worked? Does it have to do with splitting string using new line instead of some default delimiter which could be space?

**Reply by DusanLesan** [reply]
Exactly. Now all your future commands will use newline instead of space as a field separator unless you specify otherwise
